pkg/schemadsl/lexer/lex_def.go

```diff
@@ -50,11 +50,30 @@ const (
 	TokenTypeHash       // #
 	TokenTypeEllipsis   // ...
 	TokenTypeStar       // *
+
+	// Additional tokens for CEL: https://github.com/google/cel-spec/blob/master/doc/langdef.md#syntax
```

Comment:
Why do you need to emit tokens for CEL stuff?

Reply:
So the lexer can lex them. If we don't have these tokens lexed, then the parser cannot consume them, even if they aren't used directly.

Reply:
Right, but do we need strictly all the individual tokens as per CEL grammar, or could we issue the whole expression as a token? Can the lexer wait for the next `}` and issue "TokenCELExpression`?